ansible/group_vars/do/main.yml

```diff
@@ -15,7 +15,7 @@ k8s_name: "{{ project_name }}"
 bucket_name: "{{ project_name }}"
 bucket_path: "crawls"
 
-domain: "browsertrix.cloud"
+domain: "{{ domain_name | default('browsertrix.cloud') }}"
```

Comment:
Any difference between adding a separate domain_name variable instead of just overriding domain with `-e domain="yourdomain.example.com"`?